app/views/results/common/_annotations.js.erb

```diff
@@ -310,6 +311,12 @@
     };
   }
 
+  function get_notebook_annotation_range() {
+    const notebook_iframe = document.getElementById("notebook");
+    const target = notebook_iframe.contentWindow || notebook_iframe.contentDocument;
+    return target.getSelection().getRangeAt(0);
```

Comment:
It would be great to have some error-checking here---if nothing's selected, display the same kind of alert we give with the other file types.